app/javascript/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Greeting from './Greeting';
+
+function App() {
+  return (
+    <Router>
+      <Switch>
+        <Route path="/" exact component={Greeting} />
```

Comment:
- [ ] [REQUIRED] Well done @bhushan354 , however, the frontend (root route) is currently displaying a blank page but the other route works perfectly by displaying the random greetings I added in the seed file.

<img width="678" alt="Screenshot 2024-02-13 at 19 15 43" src="https://github.com/bhushan354/Rails-React-App-Webpack/assets/58692590/cae0ee71-0616-4152-8928-4596c518cd96">